lib/src/table/table-header/table-header.component.html

```diff
@@ -0,0 +1,19 @@
+<tr *ngIf="list && list.length">
+    <th *ngIf="selectable" class="table-checkbox" (click)="allSelectedClicked.emit()">
+        <input type="checkbox" class="form-control" [checked]="isAllSelected" />
+    </th>
+
+    <th
```

Comment:
i would suggest to add a custom className here that way each table header can be styled separately from the rest of the headers